tools/src/main/java/com/nedap/archie/flattener/OperationalTemplateCreator.java

```diff
@@ -50,6 +50,10 @@ public static void overrideArchetypeId(Archetype result, Archetype override) {
     }
 
     public void fillSlots(OperationalTemplate archetype) { //should this be OperationalTemplate?
+        if(!getConfig().isFillSlots()) {
```

Comment:
This skips:
1. removing closed archetype slots
2. replacing archetype roots with their inline copy
3. replacing complex object proxies with a copy of the referenced structure.

I think you only want to skip 2., and not the rest? 

If not, your configuration parameter needs an entirely different description, and then I would suggest three separate parameters.